infra/common/src/main/java/com/evolveum/midpoint/common/refinery/RefinedResourceSchemaImpl.java

```diff
@@ -43,159 +45,159 @@
  * @author mederly
  */
 public class RefinedResourceSchemaImpl implements RefinedResourceSchema {
+    private static final Trace LOGGER = TraceManager.getTrace(RefinedObjectClassDefinition.class);
```

Comment:
TraceManager should use RefinedResourceSchemaImpl.class instead of RefinedObjectClassDefinition.class.

Reply:
It is, based on the diff. I suspect you might be looking at an outdated commit?